Modules/GLO/include/GLO/VertexingQcTask.h

```diff
@@ -70,7 +73,12 @@ class VertexingQcTask final : public TaskInterface
   std::unordered_map<o2::MCEventLabel, int> mMapEvIDSourceID; // unordered_map counting the number of vertices reconstructed per event and source (--> MCEventLabel)
   TH1F* mNPrimaryMCEvWithVtx = nullptr;                       // event multiplicity for MC events with at least 1 vertex
   TH1F* mNPrimaryMCGen = nullptr;                             // event multiplicity for all MC events
-  TEfficiency* mVtxEffVsMult = nullptr;                       // for vertex efficiency
+<<<<<<< HEAD
```

Comment:
there are badly merged items, please have a look and fix them

Reply:
i did. thank you